mlir/lib/Target/SPIRV/Deserialization/Deserializer.cpp

```diff
@@ -1284,9 +1284,11 @@ LogicalResult spirv::Deserializer::processConstant(ArrayRef<uint32_t> operands,
         uint32_t word1;
         uint32_t word2;
       } words = {operands[2], operands[3]};
-      value = APInt(64, llvm::bit_cast<uint64_t>(words), /*isSigned=*/true);
+      value = APInt(64, llvm::bit_cast<uint64_t>(words), /*isSigned=*/true,
+                    /*implicitTrunc=*/true);
```

Comment:
Why would this truncate?

Reply:
Good point, this case can't truncate. I incorrectly copied it from the one directly below, which can. Fixed now.